slam-libraries/viam-cartographer/README.md

```diff
@@ -101,3 +98,7 @@ To run tests, after building, run the following
 ```bash
 ./scripts/test_cartographer.sh
 ```
+
+## Additional Notes
+
+Current available modes are: 2d
```

Comment:
[nit] Let's remove this again. We don't have any instructions on how to run cartographer so far in the README (since you need RDK for it and a config file) - and this information is only relevant when you try to run it.

Let's put this information into the cartographer tutorial instead.